Code/Editor/CryEdit.cpp

```diff
@@ -2901,14 +2901,7 @@ void CCryEditApp::OpenProjectManager(const AZStd::string& screen)
 {
     // provide the current project path for in case we want to update the project
     AZ::IO::FixedMaxPathString projectPath = AZ::Utils::GetProjectPath();
-#if !AZ_TRAIT_OS_PLATFORM_APPLE && !AZ_TRAIT_OS_USE_WINDOWS_FILE_PATHS
-    const char* argumentQuoteString = R"(")";
-#else
-    const char* argumentQuoteString = R"(\")";
-#endif
-    const AZStd::string commandLineOptions = AZStd::string::format(R"( --screen %s --project-path %s%s%s)",
-        screen.c_str(),
-        argumentQuoteString, projectPath.c_str(), argumentQuoteString);
+    const AZStd::string commandLineOptions = AZStd::string::format(" --screen %s --project-path %s", screen.c_str(), projectPath.c_str());
```

Comment:
update: Is this definitely equivalent? This looks like a change that could subtly break (especially on different platforms). This change has removed the `(` `)` which is a functional change (maybe some logic relies on looking for those, we don't know without doing more thorough testing). I'd be inclined to revert this unless there's a convincing reason not to.

Reply:
I have no idea how this change even got here.. investigating :)

Reply:
This is actually undoing a change I did to fix the quoting of path parameters in case they have spaces within them: https://github.com/o3de/o3de/commit/21ca3a4aea3b79b530e9abc1f0e2618aae88f05d#diff-7d63824d55526b922df5b01bb16d2b7e2a1bb4f3d11f908bca2bbc4617643447

Reply:
This could have been an unlucky bad merge then - I'd vote to just revert it 🙂

Reply:
I suspect it somehow sneaked in when I was reverting/splitting the original PR.

Reply:
I'll apply all suggestions from the comments and `amend` `push`

Reply:
Force push completed.
I've modified the suggested code a bit:
```cpp
const QString& selectedItem = QString()
```
instead of
```cpp
const QString& selectedItem = ""
```